test/components/ProjectSpec.js

```diff
@@ -1,48 +1,44 @@
 import test from 'tape'
 import React from 'react'
 
-import {
-  shallow
-} from 'enzyme'
+import { shallow } from 'enzyme'
```

Comment:
__[eslint]__
* Error - Unable to resolve path to module 'enzyme'. (import/no-unresolved)

<!-- Comment by [SideCI](https://sideci.com) -->